src/Symfony/Component/HttpKernel/Tests/DataCollector/TimeDataCollectorTest.php

```diff
@@ -29,7 +29,7 @@ public function testCollect()
 
         $c->collect($request, new Response());
 
-        $this->assertEquals(1000, $c->getStartTime());
+        $this->assertEquals(0, $c->getStartTime());
```

Comment:
That test is already covered on line 34. I wanted to keep this case as a regression that `REQUEST_TIME` is _not_ used when `REQUEST_TIME_FLOAT` is not filled in